pydantic/main.py

```diff
@@ -58,6 +58,14 @@
 _object_setattr = _model_construction.object_setattr
 
 
+class classproperty:
+    def __init__(self, function) -> None:
+        self.fget = function
+
+    def __get__(self, instance, owner) -> Any:
+        return self.fget(owner)
```

Comment:
This could benefit from proper type checking on the arguments. What happens currently when `model_computed_fields` is accessed with `mypy` and `pyright`?

Reply:
I think it should be solved since I removed this class. Thanks